tests/ut-torchapi.cc

```diff
@@ -419,5 +420,214 @@ TEST(torchapi, service_train_csvts_nbeats_gpu)
   ASSERT_EQ(ok_str, joutstr);
   rmdir(csvts_nbeats_repo.c_str());
 }
+#endif
+
+TEST(torchapi, nbeats_extract_layers_simple)
```

Comment:
I believe we need test with torch + extract + image classification model as this is the main usage for layer extraction.

Reply:
ok, almost done